dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -12,7 +12,7 @@ CommandBarFlyout::CommandBarFlyout()
 {
     __RP_Marker_ClassById(RuntimeProfiler::ProfId_CommandBarFlyout);
 
-#ifdef USE_INSIDER_SDK
+#ifdef USE_INTERNAL_SDK
```

Comment:
These APIs should be in the 18312 SDK, can you please include this under the "USE_INSIDER_SDK" ifdef?